api/src/db/migrations/versions/2024_10_28_add_opportunity_table_triggers.py

```diff
@@ -36,10 +37,10 @@
             opp_id := NEW.opportunity_id;
     END CASE;
 
-    INSERT INTO api.opportunity_search_index_queue (opportunity_id, has_update)
-    VALUES (opp_id, TRUE)
+    INSERT INTO api.opportunity_search_index_queue (opportunity_id)
+    VALUES (opp_id)
```

Comment:
You can't modify an existing migration. Once it runs in an environment, it won't run again unless we manually do migrate-down and that's more hassle than is worthwhile.

You'll need to make a new migration. Since the command was a `create or replace` already, it'll basically just be a copy of this